### DISCUSSION TITLE
Amazfit Bip 5 Released, Simulator Updated Simultaneously

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#231] [Announcements]
https://www.amazfit.com/products/amazfit-bip-5

<img width="1419" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/de9019c9-0bdb-4696-a110-910d0bb23f22">

<img width="798" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/0a577254-2b5e-46d7-a5b9-02fef7c74717">

**Comment by Changkey** [comment]
Bad link.
_链接打不开，提示` No such key`_

**Comment by Navragh2** [comment]
Hi, is BIP 3 also a Zepp OS watch? If so, what version of the OS does it run on?

**Reply by MyoungXUE** [reply]
<img width="996" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/ce76d840-4170-449f-8ab5-0c77b6248c49">

Device information is available at https://docs.zepp.com/docs/reference/related-resources/device-list/

**Comment by etafuro** [comment]
Is BIP 5 a target platform for development ?
When calling "zeus dev" seems that target platforms available are restricted to GTS/GTR. 
If I load a BIP 5 image in simulator, the "Hello World" app doesn't work - all is working properly on GTS/GTR emulators.
No errors are tracked in the console.
Please can you shed a light about this issue ?

**Reply by MyoungXUE** [reply]
Hello, the `app.json` of the Mini Program needs to be configured with the corresponding deviceSource.

As an example, add the following configuration to the hello world Mini Program's configuration.

```json
"platforms": [
        {
          "name": "gts4",
          "deviceSource": 7995648
        },
        {
          "name": "gts4-w",
          "deviceSource": 7995649
        },
        {
          "name": "bip5",
          "deviceSource": 8554401
        }
      ],
```

**Reply by james5252control** [reply]
i tried that and it still doesnt work. theres no errors too . its just broken. please help?

**Comment by rnaufal32** [comment]
sorry, wrong discussion
